src/snowflake/snowpark/modin/plugin/docstrings/groupby.py

```diff
@@ -817,7 +821,7 @@ def apply():
 
         Returns
         -------
-        applied : Snowpark pandas Series or Snowpark pandas DataFrame
+        Series or DataFrame
```

Comment:
we need to use 
```
:class:`~snowflake.snowpark.modin.pandas.Series` or :class:`~snowflake.snowpark.modin.pandas.DataFrame`
```
otherwise there is no link for `Series`.

cc @sfc-gh-vbudati

Reply:
This is true for our other class docstrings as well (series.py, base.py, etc.), so if we want to make this replacement we should do it in one go.

Reply:
Can you do it or busy on other staff?

Reply:
I can look into it tomorrow.

Reply:
Actually, it seems like within the `Series` or `DataFrame` module it automatically links: see the return type of [Series.quantile](https://docs.snowflake.com/en/LIMITEDACCESS/snowpark-pandas-api/reference/modin/pandas_api/snowflake.snowpark.modin.pandas.Series.quantile.html) and [DataFrame.sum](https://docs.snowflake.com/en/LIMITEDACCESS/snowpark-pandas-api/reference/modin/pandas_api/snowflake.snowpark.modin.pandas.DataFrame.sum.html) as examples. Not sure why GroupBy isn't linking ([tail](https://docs.snowflake.com/en/LIMITEDACCESS/snowpark-pandas-api/reference/modin/pandas_api/snowflake.snowpark.modin.pandas.groupby.DataFrameGroupBy.tail.html#snowflake.snowpark.modin.pandas.groupby.DataFrameGroupBy.tail) only links DF, for example).